app/controllers/user_controller.php

```diff
@@ -79,6 +80,37 @@ public function logout()
 
     public function index()
     {
+        $user = User::get();
+        $this->set(get_defined_vars());
+    }
+
+    public function edit()
+    {
+        $params = array(
+            'first_name' => Param::get('first_name'),
+            'last_name' => Param::get('last_name'),
+            'email' => Param::get('email')
+        );
+
+        $user = new User($params);
+        $page = Param::get('page_next', 'edit');
+ 
+        switch ($page) {
+            case 'edit':
+                break;
+            case 'edit_end':
+                try {
+                    $user->update();
+                }catch (ValidationException $e) {
```

Comment:
Same as this <a href='https://github.com/adrianneperedo/board/pull/2/files#r26185120'>comment</a> po :bow: